localization/localization_common/src/ros_timer.cc

```diff
@@ -21,15 +21,15 @@
 
 namespace localization_common {
 RosTimer::RosTimer(const std::string& timer_name) : averager_(timer_name, "time", "seconds"), start_time_(0) {}
-void RosTimer::Start() { start_time_ = ros::Time::now(); }
+void RosTimer::Start() { start_time_ = rclcpp::Clock().now(); }
```

Comment:
Here because it is a time diff I used the static, but we can also add a NodeHandle field to the function to get the time if that makes a difference. What do you think?

Reply:
I think the way you have it now is better, doesn't seem worth the trouble

Reply:
It depends on the purpose of the timer. If it's only intended to measure wallclock time, seems great as-is. But if it's ever intended to measure sim time when running in sim (might not be at 1:1 speed), the other way might be needed.

Reply:
oh, I hadn't realized this would be wallclock. yeah we probably want to get the ros time(?)